core/lib/config/src/configs/da_client/eigen.rs

```diff
@@ -1,10 +1,49 @@
+use std::str::FromStr;
+
 use serde::Deserialize;
-use zksync_basic_types::secrets::PrivateKey;
+use zksync_basic_types::{secrets::PrivateKey, Address, H160};
+
+/// Default address of the EigenDA service manager contract deployed on Holesky.
+const DEFAULT_EIGENDA_SVC_MANAGER_ADDRESS: &str = "0xD4A7E1Bd8015057293f0D0A557088c286942e84b";
 
-#[derive(Clone, Debug, Default, PartialEq, Deserialize)]
+/// Configuration for the EigenDA remote disperser client.
+#[derive(Clone, Debug, PartialEq, Deserialize)]
 pub struct EigenConfig {
-    pub rpc_node_url: String,
-    pub inclusion_polling_interval_ms: u64,
+    /// URL of the Disperser RPC server
+    pub disperser_rpc: String,
+    /// Block height needed to reach in order to consider the blob finalized
+    /// a value less or equal to 0 means that the disperser will not wait for finalization
+    pub settlement_layer_confirmation_depth: u32,
+    /// URL of the Ethereum RPC server
+    pub eigenda_eth_rpc: Option<String>,
```

Comment:
If this URL works as an ordinary Web3 provider URL, please use `SensitiveUrl` here, since it may contain sensitive parts for authentication.

Reply:
Sounds good, it was fixed here https://github.com/lambdaclass/zksync-era/pull/386